Merge_Array.rb

```diff
@@ -0,0 +1,79 @@
+=begin 
+Create a function that takes two arrays and combines them by alternatingly taking elements from each array in turn.
+
+The arrays may be of different lengths, with at least one character / digit.
+The first array will contain string characters (lowercase, a-z).
+The second array will contain integers (all positive).
+
+Examples
+merge_arrays(["a", "b", "c", "d", "e"], [1, 2, 3, 4, 5])
+➞ ["a", 1, "b", 2, "c", 3, "d", 4, "e", 5]
+
+merge_arrays([1, 2, 3], ["a", "b", "c", "d", "e", "f"])
+➞ [1, "a", 2, "b", 3, "c", "d", "e", "f"]
+
+merge_arrays(["f", "d", "w", "t"], [5, 3, 7, 8])
+➞ ["f", 5, "d", 3, "w", 7, "t", 8]
+=end
+
+=begin 
+Create a function that takes two arrays and combines them by alternatingly taking elements from each array in turn.
+
+The arrays may be of different lengths, with at least one character / digit.
+The first array will contain string characters (lowercase, a-z).
+The second array will contain integers (all positive).
+
+Examples
+merge_arrays(["a", "b", "c", "d", "e"], [1, 2, 3, 4, 5])
+➞ ["a", 1, "b", 2, "c", 3, "d", 4, "e", 5]
+
+merge_arrays([1, 2, 3], ["a", "b", "c", "d", "e", "f"])
+➞ [1, "a", 2, "b", 3, "c", "d", "e", "f"]
+
+merge_arrays(["f", "d", "w", "t"], [5, 3, 7, 8])
+➞ ["f", 5, "d", 3, "w", 7, "t", 8]
+=end
+
+def merge_arrays(array1, array2)
+  length = 0
+  arr = []
+
+  length1 =  array1.length
+  length2 =  array2.length
+
+  if length1 == length2
```

Comment:
this code is too complex. why would it matter which one is bigger than the other? the result should just be alternating code till the end of the array and then after that you don't get anything from that array?

Reply:
you can do this without the complexity. simplify.

Reply:
Sure !